### PR TITLE
docs(css): add scss composition rule

### PR DESCRIPTION
**Pull request by vitebo**

### CODE REVIEW

File: css.md [hunk 1]
```diff
@@ -97,3 +97,38 @@ Mantenha a especificidade da regra a mais baixa que conseguir.
 
 Isso faz a especificidade do CSS crescer desnecessariamente, deixando o código não escalável
 E o próprio BEM cria uma relação clara e tem o propósito de deixar a especificidade o menor possível
+
+### Composição
+
+E muito comum ver a composição de classes BEM com sass da seguinte forma:
```
Comment (AderbalFilho) [inline css.md:103]:
```suggestion
É muito comum ver a composição de classes BEM com Sass da seguinte forma:
```

File: css.md [hunk 2]
```diff
@@ -97,3 +97,38 @@ Mantenha a especificidade da regra a mais baixa que conseguir.
...
+    background-color: red;
+  }
+}
+```
+
+No entanto isso ta fora do nosso styleguide. Alguns dos motivos são:
+- Hoje ja temos uma base de código grande fora desse formato
+- Esse formato pode dificultar a manutenção para devs com menos familiaridade no front-end
+- Pela forma como é feita essa composição, é comum voce acabar aumentando a especificidade
+ erroneamente
```
Comment (AderbalFilho) [inline css.md:120]:
```suggestion
No entanto isso está fora do nosso styleguide. Alguns dos motivos são:
- Hoje já temos uma base de código grande fora desse formato;
- Esse formato pode dificultar a manutenção para devs com menos familiaridade no front-end;
- Pela forma como é feita essa composição, é comum acabar aumentando a especificidade
 erroneamente.
```

**Comment by vitebo** [comment]
@AderbalFilho valeu, eu apliquei as correcoes aqui image

**Review by AderbalFilho — APPROVED** [review]

**Review by plcosta — APPROVED** [review]

**Comment by vitebo** [comment]
`The base branch restricts merging to authorized users.`

**Pull request merged**